frontend/device/kindle/device.lua

```diff
@@ -1921,6 +1922,8 @@ else
         return KindleBasic5
     elseif pw6_set[kindle_devcode_v2] then
         return KindlePaperWhite6
+    elseif ks2_set[kindle_devcode_v2] then
+        return KindleScribe --- yes the scribe 2 is identical
```

Comment:
Three dashes is a special way to notify LDoc of special things.

```suggestion
        return KindleScribe -- Scribe 1 and 2 are identical.
```

Reply:
according to Amazon the newer Scribe is still a version one device but called New Kindle Scribe